collector/textfile.go

```diff
@@ -13,8 +13,8 @@ import (
 
 	dto "github.com/prometheus/client_model/go"
 
-	"code.google.com/p/goprotobuf/proto"
 	"github.com/golang/glog"
+	"github.com/golang/protobuf"
```

Comment:
1) Aha, I should have noticed the broken import path. Thanks for that.

2) No problem, blank line re-added.